pkg/ndctl/region.go

```diff
@@ -188,8 +188,9 @@ func (r *Region) CreateNamespace(opts CreateNamespaceOpts) (*Namespace, error) {
 		ways := uint64(C.ndctl_region_get_interleave_ways(ndr))
 		align := opts.Align * ways
 		if opts.Size%align != 0 {
-			// force-align down to block boundary. More sensible would be to align up, but then it may fail because we ask more then there is left
+			// align up to block boundary.
 			opts.Size /= align
+			opts.Size += 1
 			opts.Size *= align
 			glog.Warningf("%s: namespace size must align to interleave-width:%d * alignment:%d, force-align to %d",
```

Comment:
This shouldn't be a warning. A warning implies that something unexpected happened. But users have no knowledge about the expected alignment, therefore it is normal that this happens. Better use an info message at a lower priority.